openssl-dynamic/src/main/c/ssl.c

```diff
@@ -51,6 +51,8 @@ struct CRYPTO_dynlock_value {
 };
 
 
+#define ERR_LEN 256
```

Comment:
I didnt want to make to many changes which are not related to this feature... Let me to a followup

> Am 24.06.2016 um 02:50 schrieb Scott Mitchell <notifications@github.com>:
> 
> In openssl-dynamic/src/main/c/ssl.c:
> 
> > @@ -51,6 +51,8 @@ struct CRYPTO_dynlock_value {
> >  };
> >  
> >  
> > +#define ERR_LEN 256
> lets make this more generally available (in a shared header file) and use it everywhere ... unless we have good reason not to. s/ERR_error_string/ERR_error_string_n
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>